src/software/ai/evaluation/calc_best_shot.h

```diff
@@ -10,6 +10,372 @@
 #include "software/world/team.h"
 #include "software/world/world.h"
 
+class AngleSegment
```

Comment:
do you need this special class, or would `using std::pair<Angle, Angle> = AngleSegment` be sufficient?

Reply:
if I were to enforce north and south angles to top and bottom respectively then I think I would need AngleSegment

Reply:
fair enough, AngleSegment is pretty intuitive